heidrun/ncdhc.rb

```diff
@@ -33,24 +33,28 @@
 
     contributor :class => DPLA::MAP::Agent, 
     		:each => record.field('mods:name')
-    		  	.select { |name| name['mods:role'].map(&:value).include?('contributor') }, 
+    		  	.select { |name| name['mods:role']
+                                  .map { |v| v.value.downcase }
+                                  .include?('contributor') }, 
     		:as => :contrib do
       providedLabel contrib.field('mods:namePart')
     end
 
     creator :class => DPLA::MAP::Agent, 
     	    :each => record.field('mods:name')
-    		    .select { |name| name['mods:role'].map(&:value).include?('creator') }, 
+    		    .select { |name| name['mods:role']
+                              .map { |v| v.value.downcase }
+    		              .include?('creator') }, 
```

Comment:
This is odd. If I check this out, these are indented the same in my editor's `ruby-mode`. Can't figure out what's going on here. There shouldn't be whitespace issues.